src/TextifyBot.py

```diff
@@ -107,9 +107,6 @@ def processMention(mention):
                     mention.reply("Transcription was unable to identify any text within the image")
                 else:
                     makeReply(mention, result)
-        else:
```

Comment:
I think this else needs to be indented, not deleted. Not sure how the indentation got messed up but we still need to check if no URLs were found

Reply:
It checks twice. The first check is on line 89. This one is extra and it is also the second `else` statement so it will give an error

Reply:
got it, thanks